packages/portia/portia.1.5/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+name: "portia"
+version: "1.5"
+maintainer: "Cedric Cellier <rixed@happyleptic.org>"
+authors: "Cedric Cellier <rixed@happyleptic.org>"
+homepage: "https://github.com/rixed/portia"
+doc: "http://rixed.github.io/portia/"
+dev-repo: "git+https://github.com/rixed/portia.git"
+bug-reports: "mailto:rixed-opam@happyleptic.org"
+synopsis: "Literate Programming Preprocessor"
+description: """
+A literate programming preprocessor written in literate programming style.
+You need funnelweb (or portia) to compile it from scratch, but preprocessed
+source files are also available.
+
+    @O@<literate_quine.sh@>==@{@-
+    #!/bin/sh
+    cat README.md
+    @}
+"""
+build: [
+  [make]
+  [make "check"] {with-test}
+  [make "doc"] {with-doc}
+]
+install: [make "install"]
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "ocamlfind" {build}
+  "batteries"
+  "qtest" {with-test}
+]
+depexts: [
+  ["asciidoc"] {os-family = "debian"}
+]
+post-messages: [
+  "This package requires asciidoc to build the doc." {failure}
+]
```

Comment:
Do you have time to create a new `conf-asciidoc` package? This way the documentation would be available everywhere, maintainance would be reduced and the dependency would be removed during normal execution (non-doc).

If you don't have time I'll do it at some point

Reply:
I can not look into this in the near future but maybe in a few weeks.

Reply:
That's ok, I've just done it ^^